core/domain/html_cleaner.py

```diff
@@ -790,3 +791,108 @@ def add_caption_attr_to_image(html_string):
             image['caption-with-value'] = escape_html(json.dumps(''))
 
     return unicode(soup)
+
+
+def validate_customization_args(html_list):
+    """Validates customization arguments of Rich Text Components.
+
+    Args:
+        html_list: list(str). List of html strings to be validated.
+
+    Returns:
+        dict: Dictionary of all the invalid customisation args where
+        key is a Rich Text Component and value is the invalid html string.
+    """
+    # Importing this at the top of the file causes a circular dependency.
+    # So this is needed here.
+    from extensions.objects.models import objects # pylint: disable=relative-import
```

Comment:
Sorry I looked at `extensions/objects/models/objects.py` and I don't see where the circular dependency would come from. What import causes the circular dependency?

Reply:
Hi @AllanYangZhou I am not very sure about this one.

When I put the import in top of file and run the `html_cleaner_test`, I encounter an error:
```
AttributeError: 'module' object has no attribute 'html_cleaner_test'
```
I expected this because of some circular dependency which is created due to the import.

Reply:
Hi @AllanYangZhou,

The circular dependency is created as: 
- `objects` being imported in `html_cleaner`
- `schema_utils` being imported in `objects`
- `html_cleaner` being imported in `schema_utils`